pandas/core/ops/missing.py

```diff
@@ -168,7 +168,7 @@ def dispatch_fill_zeros(op, left, right, result):
         # Note: no need to do this for truediv; in py3 numpy behaves the way
         #  we want.
         result = mask_zero_div_zero(left, right, result)
-    elif op is op is rfloordiv:
```

Comment:
this is definitiely wrong as is, but is it currently parsed as `elif op is (op is rfloordiv):` in which case it would also evaluate to `False`?  If so, there may be a bug hidden in here

Reply:
can't seem to find a bug here. This code must work!

```
>>> def rfloordiv(left, right):
...     return right // left
...
>>> op = rfloordiv
>>>
>>> op is op is rfloordiv
True
>>>
>>> op is (op is rfloordiv)
False
>>>
>>> (op is op) is rfloordiv
False
>>>
>>> op is rfloordiv
True
>>>
>>> 1 is 1 is 1
True
>>>
>>> (1 is 1) is 1
False
>>>
>>> 1 is (1 is 1)
False
>>>
```